### DISCUSSION TITLE
Image field that allows image manipulation (cropping/resizing) of image before upload

### DISCUSSION THREAD

**Discussion by roland-d** [verbb/formie#1248] [Ideas]
Hello,

I have been searching but have not found anything that allows a user to upload an image and then modify the image to resize/crop/mirror the image before submitting this.

We are currently using this on another platform but as we are migrating to Craft with Formie we are looking for a replacement. Since I have not been able to find anything, I am posting it here as an idea for a new feature :)

**Comment by engram-design** [comment]
Yep, nothing like this available now, until we implement a more non-native file upload solution https://github.com/verbb/formie/issues/133

**Comment by engram-design** [comment]
Upon reviewing, I think this is probably out of the realm of Formie's expertise, at least right now. Happy to entertain options if there is demand. I feel like [Image Resizer](https://github.com/verbb/image-resizer) or something could take more ownership of this sort of thing
